regression/scripts/Vehicles/Boeing_737.py

```diff
@@ -47,8 +51,15 @@ def vehicle_setup():
     
  
     # envelope properties
+    vehicle.envelope.FARflag = 25
     vehicle.envelope.ultimate_load = 2.5
     vehicle.envelope.limit_load    = 1.5
+    vehicle.envelope.pos_limit_load    = 1.5
+    vehicle.envelope.neg_limit_load    = -1.5
+    vehicle.envelope.cruise_mach = np.array([0.785])
```

Comment:
As far as I understand, the datcom script requires a Mach number array

Reply:
I see. It would be better to have the V-n script convert the value to an array so that an array is not needed for a single value in the vehicle setup.